ckan/views/admin.py

```diff
@@ -60,8 +60,12 @@ def before_request():
 
 
 def index():
-    data = dict(sysadmins=[a.name for a in _get_sysadmins()])
-    return base.render(u'admin/index.html', extra_vars=data)
+    context = {u'user': g.user}
+    users = logic.get_action(u'user_list')(context, {})
```

Comment:
This is not very efficient and might cause performance issues with many users. I suggest:

1. Getting sysadmin users with a small helper function that queries de model directly

2. Replacing the UI widget with the autocomplete one that is used in the add member or add collaborator forms

https://github.com/ckan/ckan/blob/master/ckan/templates/group/member_new.html#L31